### ISSUE TITLE
Issues building

### ISSUE THREAD

**Issue by jpsim** [norio-nomura/LinuxSupportForXcode#2]
Getting these "No such file or directory" errors when following the installation instructions:

```
ValidateEmbeddedBinary /Users/jp/Applications/LinuxSupportForXcode.app/Contents/PlugIns/LinuxSupport.appex
    (running because there is no record of having run this command since the last clean)
    cd /Users/jp/Projects/LinuxSupportForXcode
    builtin-embeddedBinaryValidationUtility /Users/jp/Applications/LinuxSupportForXcode.app/Contents/PlugIns/LinuxSupport.appex -signing-cert 3A18C2A8179EC91AFE256053C3BA423C427E0470 -info-plist-path /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Info.plist

PBXCp /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/SourceKittenFramework.framework /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Frameworks/SourceKittenFramework.framework
    (running because there is no record of having run this command since the last clean)
    cd /Users/jp/Projects/LinuxSupportForXcode
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -exclude Headers -exclude PrivateHeaders -exclude Modules -resolve-src-symlinks /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/SourceKittenFramework.framework /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Frameworks
error: /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/SourceKittenFramework.framework: No such file or directory

PBXCp /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/SWXMLHash.framework /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Frameworks/SWXMLHash.framework
    (running because there is no record of having run this command since the last clean)
    cd /Users/jp/Projects/LinuxSupportForXcode
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -exclude Headers -exclude PrivateHeaders -exclude Modules -resolve-src-symlinks /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/SWXMLHash.framework /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Frameworks
error: /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/SWXMLHash.framework: No such file or directory

PBXCp /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/Yaml.framework /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Frameworks/Yaml.framework
    (running because there is no record of having run this command since the last clean)
    cd /Users/jp/Projects/LinuxSupportForXcode
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -exclude Headers -exclude PrivateHeaders -exclude Modules -resolve-src-symlinks /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/Yaml.framework /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Frameworks
error: /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/Yaml.framework: No such file or directory

** INSTALL FAILED **


The following build commands failed:
    PBXCp /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/SourceKittenFramework.framework /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Frameworks/SourceKittenFramework.framework
    PBXCp /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/SWXMLHash.framework /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Frameworks/SWXMLHash.framework
    PBXCp /Users/jp/Library/Developer/Xcode/DerivedData/LinuxSupportForXcode-duhrhsysrwammpemqyqnffnzmcyx/Build/Products/Debug/Yaml.framework /Users/jp/Applications/LinuxSupportForXcode.app/Contents/Frameworks/Yaml.framework
(3 failures)
```

Full log: https://gist.github.com/jpsim/7c59ac89e66c077a8c17c132d0bdaeb8

**Comment by norio-nomura** [comment]
Thank you for reporting!
I addressed this issue by #3. 🙇

**Comment by jpsim** [comment]
Thanks for the quick fix, and nicely done on adding Travis!
